src/wallet/crypto/pubkeyDecryptor.ts

```diff
@@ -1,25 +1,26 @@
 import crypto from "crypto";
-import nacl from "tweetnacl";
 import ed2curve from "ed2curve";
-import { X25519EncryptedData } from "./x25519EncryptedData";
+import nacl from "tweetnacl";
 import { UserPublicKey, UserSecretKey } from "../userKeys";
+import { X25519EncryptedData } from "./x25519EncryptedData";
 
 export class PubkeyDecryptor {
-    static decrypt(data: X25519EncryptedData, decryptorSecretKey: UserSecretKey): Buffer {
-        const ciphertext = Buffer.from(data.ciphertext, 'hex');
-        const edhPubKey = Buffer.from(data.identities.ephemeralPubKey, 'hex');
-        const originatorPubKeyBuffer = Buffer.from(data.identities.originatorPubKey, 'hex');
+    static async decrypt(data: X25519EncryptedData, decryptorSecretKey: UserSecretKey): Promise<Buffer> {
```

Comment:
Minor breaking change. Let's mention it in the PR description, as well, so that we don't forget to document it in the release notes.